src/couch/src/couch_debug.erl

```diff
@@ -891,6 +904,23 @@ print_table(Rows, TableSpec) ->
     ),
     ok.
 
+print_report(Report) ->
```

Comment:
I've noticed that the table specifically for `index_report`/`index_view_report`/`view_report` is too wide. So I was looking for opportunities to improve that. My first idea was to update TableSpec format to support `auto` in the column with. Turned out it is a bit complicated, and requiring a change in already implemented helper functions. So I decided it is not worth doing.

My second observation was the fact that for the above mentioned functions the first column is a set of field names. We know the field names upfront. They are:

- `id_num`
- `update_seq`
- `purge_seq`
- `reduce_funs`
- `def`
- `btree_size`
- `options`
- `signature`

The longest field name is 11 characters. So we could replace 50 with 11 (or 13). In order to do it we need a version of this function with arity 2.

Reply:
I like your approach. However I don't want to take `print_report/2` name. Since in the future I am planing to implement support for reports with more than 2 columns. In this case we would need to pass TableSpec in a second argument. Let's use your approach but name function differently.

```erlang
print_report(Report) ->
    print_report_with_id_with(Report, 50).

print_report_with_id_with(Report, IdWidth) ->
   TableSpec = [
        {IdWidth, left, info},
        {100, right, value}
    ],
    ....
```